### DISCUSSION TITLE
Sourcing nnn related vars to environment.d instead of shell config

### DISCUSSION THREAD

**Discussion by izmyname** [jarun/nnn#1958] [Q&A]
Hello. I tried to source all nnn-related variables inside $HOME/.config/environment.d/ in order to have all variables in one place. Everything works so far, except one thing - $nnn variable isn't sourced in this case, thus rendering all selection-related plugins and shared selection across nnn instances  unusable. Is there a way to externally source $nnn variable?

**Comment by N-R-K** [comment]
`$nnn` gets set by nnn itself. It's not meant to be set externally.

**Reply by izmyname** [reply]
I see. The problem is - it works, if nnn vars sourced inside a shell config, not if they are sourced in systemd environment. I was just wondering if it was possible to make it work this way.
